addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1340,8 +1339,7 @@ export class OdooEditor extends EventTarget {
             const el = closestElement(joinWith);
             const { zws } = fillEmpty(el);
             if (zws) {
-                // ZWS selection in OdooEditor is not working in current version of firefox (since v93.0)
-                setSelection(zws, 0, zws, this.isFirefox ? 0 : nodeSize(zws));
+                setSelection(zws, 0, zws, 0);
```

Comment:
I was just using the firefox fix everywhere.
But after rethinking it, we loose some advantages if we do not select the ZWS. 
It can generate glitch when keyboard navigating around ZWS, so selecting it can avoid the first of theses instances.

So I'll revert this

Reply:
After Discord discussion, we decided to keep the range collapsed. mainly so the user can always see the cursor in the document.